development-docs/DEV_GUIDE.md

```diff
@@ -19,248 +25,287 @@ This document gives a detailed breakdown of the various build processes and opti
+updated bash run `sudo bash -c 'echo /usr/local/bin/bash >> /etc/shells'` and `chsh -s /usr/local/bin/bash`
+
+The `mvn` tool might install the latest version of OpenJDK during the brew install. For builds on macOS to succeed,
+OpenJDK version 11 needs to be installed. This can be done by running `brew install openjdk@11`. For maven to read the
+new Java version, you will need to edit the `~/.mavenrc` file and paste the following
+line `export JAVA_HOME=/Library/Java/JavaVirtualMachines/openjdk-11.jdk/Contents/Home`.
+
+You may come across an issue of linking from the above step. To solve this run this
+command `sudo ln -sfn /usr/local/opt/openjdk@11/libexec/openjdk.jdk /Library/Java/JavaVirtualMachines/openjdk-11.jdk`.
+If this throws an error that it cannot find the file or directory, navigate into `/Library/Java/` (or how ever deep you
+can) and create a new folder named `JavaVirtualMachines` followed by creating a file named `openjdk-11.jdk`. The folder
+structure after everything is said and done should look like `/Library/Java/JavaVirtualMachines/openjdk-11.jdk`. After
+doing that run the command at the beginning again and this should link the file and allow you to use maven with OpenJDK
+version 11.
+
+### Kubernetes or OpenShift Cluster
+
+In order to run the integration tests and test any changes made to the operators you will need a functioning Kubernetes
+or OpenShift cluster. This can be a remote cluster or a local development cluster.
 
-To build Strimzi from a source the operator and Kafka code needs to be compiled into Docker container images and placed in a location accessible to the Kubernetes/OpenShift nodes. The easiest way to make your personal Strimzi builds accessible, is to place them on the [Docker Hub](https://hub.docker.com/). The instructions below use this method, however other build options (including options for limited or no network access) are available in the sections below this quick start guide.
+#### Minikube
+
+The default Minikube setup should allow the integration tests to be run without additional configuration changes.
+
+Sometimes however, updates to Minikube may prevent the test cluster context being correctly set. If you have errors due
+to `oc` commands being called on your Minikube cluster then explicitly set the `TEST_CLUSTER` environment variable
+before running the `make` commands.
+
+    export TEST_CLUSTER=minikube
 
-1. If you don't have one already, create an account on the [Docker Hub](https://hub.docker.com/). Then log your local Docker client into the Hub using:
+## Build and deploy from source
+
+To build Strimzi from a source the operator and Kafka code needs to be compiled into Docker container images and placed
+in a location accessible to the Kubernetes/OpenShift nodes. The easiest way to make your personal Strimzi builds
+accessible, is to place them on the [Docker Hub](https://hub.docker.com/). The instructions below use this method,
```

Comment:
```suggestion
accessible, is to place them on the public container registry like [Docker Hub](https://hub.docker.com/) or quay (link). The instructions below use Docker Hub,
```

Reply:
Thanks @Frawless have pushed a new commit incorporating all your suggestions, plus a couple of other slight issues I found.